Utilities/Atomic.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
```

Comment:
This and the other file should be in `Sources/ApolloWebSocket` so they're included with Swift Package Manager.

Alternately it could go in `Sources/Apollo` if you make the class `public` - I have a feeling this may be a good replacement for some other locking code we have that's not quite working the way we want it to.

Reply:
Oops, wasn't aware of the folder structure. I'll go with the first option for now.